src/App.css

```diff
@@ -2,9 +2,38 @@
   box-sizing: border-box;
 }
 
+:root {
+  --primary-dark: rgb(0, 16, 46);
+  --primary: rgb(23, 42, 79);
+  --primary-light: rgb(24, 128, 177);
+  --primary-very-light: rgb(211, 221, 245);
+
+  --secondary: rgb(41, 162, 71);
+  --secondary-light: rgb(158, 209, 158);
+
+  --neutral-very-dark: rgb(38, 38, 38);
+  --neutral-dark: rgb(177, 177, 177);
+  --neutral: rgb(206, 206, 206);
+  --neutral-light: rgb(238, 238, 238);
+  --neutral-very-light: rgb(255, 255, 255);
```

Comment:
you can user white black and grey names - don't change this now, just keep in mind :)